flutter_local_notifications/ios/Classes/FlutterLocalNotificationsPlugin.m

```diff
@@ -234,6 +268,16 @@ - (void)pendingNotificationRequests:(FlutterResult _Nonnull)result {
   }
 }
 
+- (void)getActiveNotifications:(FlutterResult _Nonnull)result {
+  if (@available(iOS 10.0, *)) {
+    [self activeUserNotificationRequests:result];
+  } else {
+    NSMutableArray<NSMutableDictionary<NSString *, NSObject *> *>
+      *activeNotifications = [[NSMutableArray alloc] initWithCapacity:0];
+    result(activeNotifications);
```

Comment:
I have found the `FlutterError` in this file. Could you help me what parameters to use in this case for `code`, `message`, and `details`?

Reply:
Sure

- Currently on Android, code is `GET_ACTIVE_NOTIFICATIONS_ERROR_CODE` https://github.com/MaikuB/flutter_local_notifications/blob/a5a84f095a6e7a0d193a8acfc747427a9ff867d4/flutter_local_notifications/android/src/main/java/com/dexterous/flutterlocalnotifications/FlutterLocalNotificationsPlugin.java#L124. In hindsight, this could be done better to be generic error code on both Android, iOS and macOS (e.g. `OS_VERSION_NOT_SUPPORTED`. Options here are to update it, including on Android or you can iOS and macOS consistent and I can update it later
- `message` would be similar to what's on Android, which is currently `Android version must be 6.0 or newer to use getActiveNotifications`. Replace `Android` with `iOS`, `macOS` and then the version number with the appropriate OS version
- `details` can be omitted as it should be an optional parameter